src/Layout/layout-styles.js

```diff
@@ -0,0 +1,135 @@
+// @flow weak
+import { reduce, range, flow, partialRight } from 'lodash/fp';
+
+const baselineGrid = 8;
+
+const generateProps = (propSetter, from, to) => flow(
+  range,
+  reduce((a, i) => ({
+    ...a,
+    ...propSetter(i),
+  }), { }),
+)(from, to);
+
+const flexOrder = () => ({
+  flexOrder: { order: 0 },
+  ...generateProps((i) => ({ [`flexOrder${i}`]: { order: i } }), -20, 20),
```

Comment:
`[-20,20]` is quite a big range. Would using inline-style for this be a good approach? Or using a smaller window? I know that angular/material is using this value.